pkg/proc/variables.go

```diff
@@ -1026,13 +1031,13 @@ func (v *Variable) loadValueInternal(recurseLevel int, cfg LoadConfig) {
 //   is performed.
 // * If srcv and dstv have the same type and are both addressable then the
 //   contents of srcv are copied byte-by-byte into dstv
-func (dstv *Variable) setValue(srcv *Variable, srcExpr string) error {
+func (v *Variable) setValue(srcv *Variable, srcExpr string) error {
```

Comment:
I would prefer that this remained `dstv` for symmetry with `srcv`.

Reply:
I'll change if you feel strongly, but I like the consistency of receiver names for these methods.

Reply:
ok, leave it